pycuda/gpuarray.py

```diff
@@ -252,6 +257,24 @@ def __init__(
 
         self._grid, self._block = splay(self.mem_size)
 
+    @property
+    def __cuda_array_interface__(self):
+        """Returns a CUDA Array Interface dictionary describing this array's
+        data."""
+        if self.gpudata is not None:
+            ptr = int(self.gpudata)
+        else:
+            ptr = 0
+
+        return {
+            "shape": self.shape,
+            "strides": self.strides,
+            "data": (ptr, False),
```

Comment:
Could you add a comment explaining the `False`?

Reply:
Will do.

Reply:
Comment added.